index.js

```diff
@@ -35,10 +30,23 @@ const contacts = [
   "Will Faurot"
 ]
 
-function listContacts() {
+$(document).ready(function(){
+  listContacts(contacts)
+  sortByFirstName()
+})
+
+
+function listContacts(contacts) {
```

Comment:
Re: making functions pure:

This function is a good example. It's job is to take some data (contacts) and append information to the DOM. It does a single thing (no side effects), and it does the thing you would expect. 👍